code/modules/mob/living/carbon/human/appearance.dm

```diff
@@ -3,20 +3,20 @@
 	AC.flags = flags
 	AC.ui_interact(user, state = state)
 
-/mob/living/carbon/human/proc/change_species(var/new_species)
-	if(!new_species || species == new_species || !(new_species in all_species))
-		return
+/mob/living/carbon/human/proc/change_species(datum/species/new_species)
+	if(!new_species || dna.species.type == new_species || !(initial(new_species.name) in all_species))
```

Comment:
Is that third check sensical? Would it be better to do `!ispath(new_species, /datum/species)` here?